src/walletSdk/Utils/soroban.ts

```diff
@@ -165,3 +165,83 @@ export const parseTokenAmount = (
 
   return BigInt(parsed.toString());
 };
+
+// This function attempt to convert soroban contract values
```

Comment:
can we use the jsdoc comment style for describing what the method's doing? [for example ](https://github.com/stellar/typescript-wallet-sdk/blob/e9943d82d28e5a6206c5636a4e53e93b4dc3e8f1/%40stellar/typescript-wallet-sdk/src/walletSdk/Anchor/Sep6.ts#L98-L115)

Reply:
and actually if we could do for all of the methods in this file 🙏 

will keep it consistent with other code in the codebase, and helps users understand how to use the methods

Reply:
for sure!